### ISSUE TITLE
move under tests/ folder

### ISSUE THREAD

**Issue by plika** [deefactorial/jxcore-cordova-tests#1]
@deefactorial @obastemur I'm suggesting moving this repo under test/ folder on jxcore-cordova project. One question though, can't we use this also for iOS tests?

**Comment by obastemur** [comment]
I couldn't look into this yet but sounds like a plan.

**Comment by deefactorial** [comment]
Yes theoretically it could work under ios I don't have the platform to test. Moving it to the test folder of jxcore-cordova is the plan.

**Comment by obastemur** [comment]
Okay I will try to test it on weekend. @deefactorial thanks for the great job!
